### ISSUE TITLE
Integration is no longer loaded. Error after update from 0.0.4 to 0.0.9

### ISSUE THREAD

**Issue by mathiaslehmann** [mikey0000/Mammotion-HA#53]
### The problem

Hello, I get the following error after updating from version 0.0.4 to 0.0.9.

```
2024-08-08 08:47:06.626 ERROR (SyncWorker_14) [homeassistant.util.package] Unable to install package pymammotion==0.0.45: error: subprocess-exited-with-error
2024-08-08 08:47:16.751 ERROR (SyncWorker_14) [homeassistant.util.package] Unable to install package pymammotion==0.0.45: error: subprocess-exited-with-error
2024-08-08 08:47:26.722 ERROR (SyncWorker_14) [homeassistant.util.package] Unable to install package pymammotion==0.0.45: error: subprocess-exited-with-error
2024-08-08 08:47:26.723 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration 'mammotion': Requirements for mammotion not found: ['pymammotion==0.0.45'].
homeassistant.requirements.RequirementsNotFound: Requirements for mammotion not found: ['pymammotion==0.0.45'].
```

### What Mammotion mower are you using?

Yuka

### What version of the Mammotion intergration are you using?

f494592 v0.0.9

### What version of Home Assistant Core are you running?

core-2024.8.0

**Comment by mikey0000** [comment]
Can you pull down the latest version, I think it's fixed in 0.0.10

8/08/2024 7:10:03 pm mathiaslehmann ***@***.***>:

**Comment by mathiaslehmann** [comment]
Hello, I have just updated to version 0.0.10. Thanks for the quick adjustment. The integration is loading again. Unfortunately there are no more entities. I deleted the device and it was found again. Unfortunately there are no entities after setting it up either.

**Comment by mikey0000** [comment]
I'll be looking at it in the morning my time.

**Comment by mathiaslehmann** [comment]
Okay, thank you very much for your effort.

**Comment by mikey0000** [comment]
Resolved in 0.0.12
